### DISCUSSION TITLE
有宏基因组reads的fastq_1和_2文件，有功能基因的数据库，想要直接比对得到功能基因与样本的OTU表可以用什么软件呢？

### DISCUSSION THREAD

**Discussion by tiantianlili** [ShixiangWang/self-study#40] [Q&A] [answered]
如题，刚接触宏基因组类型的分析很多问题不懂，看到了公众号推荐就来这里提问了。尝试用过blast不过好像不能以_1和_2文件作为输入，想直接以reads作为输入有大佬能推荐软件进行比对嘛？？在此先行谢过~~

**Comment by ShixiangWang** [comment, marked as answer]
@tiantianlili 我没有分析宏基因组的经验，如果你是初学者，我建议你通过谷歌搜索宏基因组分析教程，另外学术方面也可以通过谷歌找一些综述性的文章逐步学习。

例如下面就是我搜索的一个内容，展示了宏基因组大致的流程。

![](http://www.guhejk.com/wordpress/wp-content/uploads/2020/04/11-1-1024x744.png)

> 来源：http://www.guhejk.com/wordpress/?p=1664

这里有一系列文章你可以参考：https://blog.csdn.net/woodcorpse/category_9274491.html

这里有很多资讯：https://www.mr-gut.cn/discovery/recommend/1188098161

目前有很多专门写宏基因分析的公众号（宏基因组应该比较有名），现在资源太多了

例如：https://weixin.sogou.com/weixin?type=2&s_from=input&query=%E5%AE%8F%E5%9F%BA%E5%9B%A0%E7%BB%84&ie=utf8&_sug_=n&_sug_type_=

你需要做的是通过搜索引擎找到一些专业的教程，然后照着学习，然后同时寻找和了解你研究内容相关的工具和方法。

最后分享一个我写的浏览器搜索插件，可以比较快地搜索公众号内容：https://github.com/ShixiangWang/r-search-extension

**Comment by tiantianlili** [comment]
> @tiantianlili我没有分析宏基因组的经验，如果你是初学者，我建议你通过谷歌搜索宏基因组分析教程，其他学术方面也可以通过谷歌找一些概括性的文章逐步学习。
> 
> 例如下面就是我搜索的一个内容，展示了宏基因组大致的流程。
> 
> ![](https://camo.githubusercontent.com/9eabf9236b4899e79a9d31a2275c4b7f48f2fc3e/687474703a2f2f7777772e677568656a6b2e636f6d2f776f726470726573732f77702d636f6e74656e742f75706c6f6164732f323032302f30342f31312d312d31303234783734342e706e67)
> 
> > 来源：[http](http://www.guhejk.com/wordpress/?p=1664) : [//www.guhejk.com/wordpress/?](http://www.guhejk.com/wordpress/?p=1664) p [=](http://www.guhejk.com/wordpress/?p=1664) 1664[](http://www.guhejk.com/wordpress/?p=1664)
> 
> 这里有各种各样的文章你可以参考：[https](https://blog.csdn.net/woodcorpse/category_9274491.html) ://blog.csdn.net/woodcorpse/category_9274491.html[](https://blog.csdn.net/woodcorpse/category_9274491.html)
> 
> 这里有很多资讯：[https](https://www.mr-gut.cn/discovery/recommend/1188098161) : [//www.mr-gut.cn/discovery/recommend/1188098161](https://www.mr-gut.cn/discovery/recommend/1188098161)
> 
> 目前有很多专门写宏基因分析的公众号（宏基因组应该比较有名），现在资源太多了
> 
> 例如：[https](https://weixin.sogou.com/weixin?type=2&s_from=input&query=%E5%AE%8F%E5%9F%BA%E5%9B%A0%E7%BB%84&ie=utf8&_sug_=n&_sug_type_=) ://weixin.sogou.com/weixin [?](https://weixin.sogou.com/weixin?type=2&s_from=input&query=%E5%AE%8F%E5%9F%BA%E5%9B%A0%E7%BB%84&ie=utf8&_sug_=n&_sug_type_=) type [=2&](https://weixin.sogou.com/weixin?type=2&s_from=input&query=%E5%AE%8F%E5%9F%BA%E5%9B%A0%E7%BB%84&ie=utf8&_sug_=n&_sug_type_=) s_from [=](https://weixin.sogou.com/weixin?type=2&s_from=input&query=%E5%AE%8F%E5%9F%BA%E5%9B%A0%E7%BB%84&ie=utf8&_sug_=n&_sug_type_=) input [&](https://weixin.sogou.com/weixin?type=2&s_from=input&query=%E5%AE%8F%E5%9F%BA%E5%9B%A0%E7%BB%84&ie=utf8&_sug_=n&_sug_type_=) query [=%](https://weixin.sogou.com/weixin?type=2&s_from=input&query=%E5%AE%8F%E5%9F%BA%E5%9B%A0%E7%BB%84&ie=utf8&_sug_=n&_sug_type_=) E5%AE%8F%E5%9F%BA%E5%9B%A0%E7%BB%84 [&](https://weixin.sogou.com/weixin?type=2&s_from=input&query=%E5%AE%8F%E5%9F%BA%E5%9B%A0%E7%BB%84&ie=utf8&_sug_=n&_sug_type_=) ie [=](https://weixin.sogou.com/weixin?type=2&s_from=input&query=%E5%AE%8F%E5%9F%BA%E5%9B%A0%E7%BB%84&ie=utf8&_sug_=n&_sug_type_=) utf8 [&](https://weixin.sogou.com/weixin?type=2&s_from=input&query=%E5%AE%8F%E5%9F%BA%E5%9B%A0%E7%BB%84&ie=utf8&_sug_=n&_sug_type_=) _sug_ [=n&](https://weixin.sogou.com/weixin?type=2&s_from=input&query=%E5%AE%8F%E5%9F%BA%E5%9B%A0%E7%BB%84&ie=utf8&_sug_=n&_sug_type_=) _sug_type_ [=](https://weixin.sogou.com/weixin?type=2&s_from=input&query=%E5%AE%8F%E5%9F%BA%E5%9B%A0%E7%BB%84&ie=utf8&_sug_=n&_sug_type_=)
> 
> 您需要做的是通过搜索引擎找到一些专业的教程，然后照着学习，然后同时寻找和了解您研究内容相关的工具和方法。
> 
> 最后分享一个我写的浏览器搜索插件，可以比较快地搜索公众号内容：[https](https://github.com/ShixiangWang/r-search-extension) : [//github.com/ShixiangWang/r-search-extension](https://github.com/ShixiangWang/r-search-extension)

**Comment by tiantianlili** [comment]
好的好的，非常感谢~~
